app/assets/javascripts/discourse/app/controllers/auth-token.js

```diff
@@ -1,48 +0,0 @@
-import Controller from "@ember/controller";
-import ModalFunctionality from "discourse/mixins/modal-functionality";
-import { ajax } from "discourse/lib/ajax";
-import { action } from "@ember/object";
-import { next } from "@ember/runloop";
-import { userPath } from "discourse/lib/url";
-
-export default Controller.extend(ModalFunctionality, {
-  expanded: false,
-
-  onShow() {
-    ajax(
-      userPath(`${this.get("currentUser.username_lower")}/activity.json`)
-    ).then((posts) => {
-      if (posts.length > 0) {
-        this.set("latest_post", posts[0]);
-        // slightly hacky, but default d-modal focus gets reset
-        document.querySelector(".d-modal .modal-close")?.focus();
-      }
-    });
-  },
-
-  @action
-  toggleExpanded(event) {
-    event?.preventDefault();
-    this.set("expanded", !this.expanded);
-  },
-
-  actions: {
-    highlightSecure() {
-      this.send("closeModal");
-
-      next(() => {
-        const $prefPasswordDiv = $(".pref-password");
-
-        $prefPasswordDiv.addClass("highlighted");
-        $prefPasswordDiv.on("animationend", () =>
-          $prefPasswordDiv.removeClass("highlighted")
-        );
-
-        window.scrollTo({
-          top: $prefPasswordDiv.offset().top,
-          behavior: "smooth",
-        });
-      });
```

Comment:
I have removed this logic and related tests. It is 
1. Not necessary
2. Not actually working / doing anything
- We don't even have a class of `highlighted` that does anything
- We have scroll logic for moving you to the password button but there is little to no content on this user preference page, so you never engage with the scrolling, even on mobile.